config/environments/development.rb

```diff
@@ -86,4 +86,6 @@
     authentication: :plain,
     enable_starttls_auto: true
   }
+
```

Comment:
What about production.rb? I guess, You should add the same line to it.

Reply:
Added.